common/static/coffee/spec/discussion/view/discussion_thread_list_view_spec.coffee

```diff
@@ -6,17 +6,17 @@ describe "DiscussionThreadListView", ->
         appendSetFixtures("""
         <script type="text/template" id="thread-list-template">
             <div class="forum-nav-header">
-                <a href="#" class="forum-nav-browse" aria-haspopup="true">
-                    <i class="icon fa fa-bars"></i>
+                <button type="button" class="forum-nav-browse" id="forum-nav-browse" aria-haspopup="true">
+                    <span class="icon fa fa-bars" aria-hidden="true"></span>
                     <span class="sr">Discussion topics; current selection is: </span>
                     <span class="forum-nav-browse-current">All Discussions</span>
                     ▾
-                </a>
+                </button>
                 <form class="forum-nav-search">
                     <label>
                         <span class="sr">Search</span>
-                        <input class="forum-nav-search-input" type="text" placeholder="Search all posts">
-                        <i class="icon fa fa-search"></i>
+                        <input class="forum-nav-search-input" id="forun-nav-search" type="text" placeholder="Search all posts">
```

Comment:
not sure if you are using this id for anything important but there is a typo: forun